src/compiler/crystal/tools/formatter.cr

```diff
@@ -985,7 +985,9 @@ module Crystal
         StringLiteral.new(name).accept self
       else
         write @token
+        @lexer.wants_symbol = false
```

Comment:
Using `next_token_never_a_symbol` should be the same, right?

Reply:
Almost: `next_token` is overwritten in the Formatter so that method doesn't exist there. And there's no `next_token_never_a_symbol` in the Formatter, only in the lexer.

Reply:
roger, ok.